src/utils.js

```diff
@@ -2,12 +2,14 @@ import { cancelled, put, take } from 'redux-saga/effects'
 
 export const noop = x => x
 
-export function * syncChannel (channel, successActionCreator, transform = noop) {
+export function * syncChannel (channel, successActionCreator, transform = noop, failureActionCreator = null) {
```

Comment:
I'm not sure about the order of the arguments here. Do you think `transform` is more useful than `failureActionCreator`?
Maybe this should be an object with three keys to make things simpler.